client/post-editor/media-modal/gallery/fields.jsx

```diff
@@ -144,13 +151,26 @@ export default React.createClass( {
 		);
 	},
 
+	renderReverseOrderButton() {
+		const { translate } = this.props;
+
+		return (
+			<EditorMediaModalFieldset>
+				<Button onClick={ this.updateReverseOrder } disabled={ this.props.settings.orderBy === 'rand' } >
+					{ translate( 'Reverse Order' ) }
+				</Button>
+			</EditorMediaModalFieldset>
+		);
+	},
+
 	render() {
 		const types = this.getTypeOptions();
 		const links = this.getLinkOptions();
 		const sizes = this.getSizeOptions();
 
 		return (
 			<div className="editor-media-modal-gallery__fields">
+				{ this.renderReverseOrderButton() }
 				{ this.renderDropdown( this.translate( 'Layout' ), types, 'type' ) }
```

Comment:
I might recommend that you consider updating `this.translate` to `this.props.translate` in the same pull request if you're introducing `localize`.